src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -19,7 +22,13 @@ const pageTypeUrls = (assetType, service, variant) => {
           name: 'secondaryColumn',
           path: getSecondaryColumnUrl({ service, variant }),
         },
-      ];
+        hasRecommendations(service, variant)
+          ? {
+              name: 'recommendations',
+              path: getRecommendationsUrl({ assetUri, variant }),
+            }
+          : null,
+      ].filter(i => i);
```

Comment:
does this filter out null values ?

Reply:
Yep https://github.com/bbc/simorgh/pull/7231#discussion_r456310457